locale/de/config.cfg

```diff
@@ -0,0 +1,87 @@
+[entity-name]
+gunship-airborne=Kanonenboot
+cargo-plane-airborne=Frachtflugzeug
+jet-airborne=Jet
+flying-fortress-airborne=Fliegende Festung
+better-cargo-plane-airborne=Besseres Frachtflugzeug
+even-better-cargo-plane-airborne=Noch besseres Frachtflugzeug
+
+[entity-description]
+gunship-airborne=Aufklärungs- und Kampfflugzeug
+cargo-plane-airborne=Eine fliegende Langstrecken-Truhe.
+jet-airborne=Wenn du SCHNELL an einen Ort gelangen musst.
+flying-fortress-airborne=Der ultimative Luftüberlegenheitsflieger.
+better-cargo-plane-airborne=Ein bessere fliegende Langstrecken-Truhe.
+even-better-cargo-plane-airborne=Ein noch bessere fliegende Langstrecken-Truhe.
+
+[mod-setting-name]
+aircraft-realism-turn-radius=Realistischer Wenderadius
+aircraft-realism-acceleration=Realistische Beschleunigung
+aircraft-realism-braking-speed=Realistische Bremskraft
+aircraft-realism-takeoff-health=Erhöhter Bodenkollisionsschaden
+aircraft-realism-no-aerial-shadows=Flugzeugschatten

```

Comment:
Should be **NO** plane shadow.
I changed this in the locale for English for 1.3 since I recognized it is incorrect.